marketing/signals.py

```diff
@@ -21,27 +21,22 @@
 """
 from django.dispatch import receiver
 
-from instance.signals import appserver_spawned
+from registration.signals import betapplication_accepted
```

Comment:
Rename the signal `betatestpplication_accepted` for consistency with model names and other references in the code.
```suggestion
from registration.signals import betatestpplication_accepted
```

Reply:
Done.